Makefile

```diff
@@ -333,12 +331,19 @@ reconfigure:
 	./configure $(CONFIGURE_ARGS)
 endif
 
+utils/domainstate.ml: utils/domainstate.ml.c runtime/caml/domain_state.tbl
```

Comment:
These utils/domainstate.ml and utils/domainstate.mli need to go in `.gitignore`

Reply:
Done.